scripts/mockGuardianSpy.ts

```diff
@@ -0,0 +1,356 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+import { Server, ServerCredentials } from "@grpc/grpc-js";
+import { ServerWritableStreamImpl } from "@grpc/grpc-js/build/src/server-call";
+
+import { ethers } from "ethers";
+import { RetryProvider } from "../packages/package-utils";
+
+import {
+  FilterEntry,
+  SpyRPCServiceService,
+  SubscribeSignedVAARequest,
+  SubscribeSignedVAAResponse,
+} from "../lib/wormhole/sdk/js-proto-node/src/spy/v1/spy";
+import { FORKED_XDAI_CHAINID } from "../helpers/constants";
+
+// Random key
+
+import { abi as bridgeAbi } from "../artifacts/contracts/testHelpers/WormholeMock.sol/WormholeMock.json";
+import { abi as wormholeBridgeForColonyAbi } from "../artifacts/contracts/bridging/WormholeBridgeForColony.sol/WormholeBridgeForColony.json";
+
+function ethereumAddressToWormholeAddress(address: string) {
+  return ethers.utils.hexZeroPad(ethers.utils.hexStripZeros(ethers.utils.hexlify(address)), 32);
+}
+
+type QueueEntry = [ethers.Contract, string, number, number, string, number, number];
+
+class MockGuardianSpy {
+  homeRpc: string;
+
```

Comment:
Are these blank lines necessary? It's a lot of extra vertical space

Reply:
It's being enforced by https://eslint.org/docs/latest/rules/lines-between-class-members which is coming from [eslint-config-colony](https://www.npmjs.com/package/@colony/eslint-config-colony).